install/windows/Makefile

```diff
@@ -0,0 +1,68 @@
+include ../.env
+BINVER=windows_amd64
+INSTALLFORMAT=x86_64.msi
+VIRTUALBOX_FULLNAME=VirtualBox-6.1.12-139181-Win.exe
```

Comment:
This shouldn't be hardcoded but instead picked up from .env file.

Reply:
Yes- I'll create a separate issue so we can do this when testing in windows as well.